string name
string description
int8 vtype
int8 ltype
TobiIcClass[] classes
